scripts/linux/electron-installer-appimage.sh

```diff
@@ -143,12 +144,19 @@ else
 fi
 
 # Generate AppImage
+mkdir -p "$(dirname "$ARGV_OUTPUT")"
 rm -f "$ARGV_OUTPUT"
 
 APPIMAGEASSISTANT_PATH=/tmp/AppImageAssistant.AppImage
 download_executable \
   "$APPIMAGES_GITHUB_RELEASE_BASE_URL/AppImageAssistant_$APPIMAGES_TAG-$APPIMAGES_ARCHITECTURE.AppImage" \
   $APPIMAGEASSISTANT_PATH
 
-$APPIMAGEASSISTANT_PATH "$APPDIR_PATH" "$ARGV_OUTPUT"
+$APPIMAGEASSISTANT_PATH "$APPDIR_PATH" "$(dirname "$ARGV_OUTPUT")/$OUTPUT_FILENAME"
```

Comment:
This script assumes that `OUTPUT_FILENAME` has a specific format, and is always in the same directory as `$(dirname $ARGV_OUTPUT)`. Might be better to instead have an extra command-line option to this script giving the full path to `OUTPUT_FILENAME`?
(And perhaps `OUTPUT_FILENAME` isn't the best name for this parameter, given that it's effectively an 'input' to this script?)